samigo/samigo-app/src/webapp/js/authoring.js

```diff
@@ -389,16 +400,19 @@ function disableAllFeedbackCheckTemplate(feedbackType)
 	}
 }
 
-$(document).ready(function(){
+$(window).load( function() {
```

Comment:
So document.ready runs when the DOM is ready but window.load is when the whole page is loaded including the content.

I noticed much smoother transitions while using window.load vs document.ready. Also noticed some javascript issues while updating the date pickers state.